packages/ui/src/components/va-input/components/VaInputWrapper.vue

```diff
@@ -74,10 +74,10 @@
       </div>
     </div>
 
-    <div v-if="$props.counter" class="va-input-wrapper__bottom">
-      <slot name="counter" v-bind="{ valueLength: $props.valueLength, maxLength: $props.maxLength }">
+    <div v-if="isCounterVisible" class="va-input-wrapper__bottom">
+      <slot name="counter" v-bind="{ valueLength: $props.counterValue, maxLength: $props.maxLength }">
         <div class="va-input-wrapper__bottom-counter">
-          {{ $props.maxLength ? `${valueLength}/${$props.maxLength}` : valueLength }}
+          {{ $props.maxLength ? `${$props.counterValue}/${$props.maxLength}` : $props.counterValue }}
```

Comment:
maybe variable?
```ts
counterString = computed(() => props.maxLength !== undefined
  ? `${props.counterValue}/${props.maxLength}`
  : String(props.counterValue)
)
```

Reply:
I think it's not that necessary, because the counter is `slot` and can be replaced with the custom user content.